rpm/telepathy-farstream.spec

```diff
@@ -9,9 +9,10 @@ Source0:        http://telepathy.freedesktop.org/releases/telepathy-farstream/%{
 Patch0:         disable-gtkdoc.patch
 BuildRequires:  python
 BuildRequires:  pkgconfig(telepathy-glib)
-BuildRequires:  pkgconfig(farstream-0.1)
+BuildRequires:  pkgconfig(farstream-0.2)
 BuildRequires:  pkgconfig(dbus-1)
 BuildRequires:  pkgconfig(dbus-glib-1)
+BuildRequires:  pkgconfig(gobject-introspection-1.0)
```

Comment:
Does it still require introspection if you're building with `--disable-introspection`?